docs/articles/documentation/reference/test-api/testcontroller/openwindow.md

```diff
@@ -4,12 +4,12 @@ title: t.openWindow Method
 permalink: /documentation/reference/test-api/testcontroller/openwindow.html
 ---
 
-# t.openWindow method
+# t.openWindow Method
 
-Opens a new browser window. Returns the matching window descriptor and can be chained with other TestController methods.
+Opens a new browser window. Returns the matching window descriptor and can be chained with other `TestController` methods.
 
-```js
-t.openWindow( url ) → Object
+```text
+t.openWindow( url ) → this | Promise<unknown>
```

Comment:
`this | Promise<any>`?

Reply:
Perhaps I'm wrong. Here's the source:

[ openWindow(url: string): WindowDescriptorPromise;](https://github.com/DevExpress/testcafe/blob/afb1e857987cb2623c23529237716b34fb42bb5c/ts-defs-src/test-api/test-controller.d.ts#L308)
[interface WindowDescriptorPromise extends TestController, Promise<WindowDescriptor> {}](https://github.com/DevExpress/testcafe/blob/afb1e857987cb2623c23529237716b34fb42bb5c/ts-defs-src/test-api/test-controller.d.ts#L420)
[type WindowDescriptor = unknown;](type WindowDescriptor = unknown;)